README.md

```diff
@@ -29,6 +29,7 @@ Model registry provides a central repository for model developers to store and m
  - [Logical Model](./docs/logical_model.md)
 4. Python client
  - [installing and using the Model Registry Python client](https://model-registry.readthedocs.io/en/latest/)
+ - [Python client development](clients/python/README.md)
```

Comment:
I don't think we need the latter as they're the same :)

<img width="1920" height="1080" alt="Screenshot 2025-08-05 at 22 02 50 (2)" src="https://github.com/user-attachments/assets/7889d8aa-38e9-463e-b0aa-08193b1b11eb" />

Reply:
👍 removed the link